.github/workflows/release.yaml

```
@@ -77,7 +77,7 @@ jobs:
         name: python-package-distributions
         path: dist/
     - name: Sign the dists with Sigstore
-      uses: sigstore/gh-action-sigstore-python@v2.1.1
+      uses: sigstore/gh-action-sigstore-python@v3
```

I think you have to specify the full version here:

```suggestion
      uses: sigstore/gh-action-sigstore-python@v3.0.0
```

The tag v3 does not exist in this action repo.
See https://github.com/sigstore/gh-action-sigstore-python/tags